test/DynamicDependency/data/DynamicDependencyLifetimeManagerGC1010.Msix/appxmanifest.xml

```diff
@@ -45,7 +45,7 @@
       </uap:VisualElements>
       <Extensions>
         <uap3:Extension Category="windows.appExtension">
-          <uap3:AppExtension Name="com.microsoft.reunion.ddlm.4.1.###architecture###"
+          <uap3:AppExtension Name="com.microsoft.reunion.ddlm-4.1-###architecture###"
```

Comment:
There is alot of tidbits like this I am not understanding in general but I trust it's there for a reason. Like why is there a hardecoded 4.1 in the name?

Reply:
These are test packages. They use constant data. Real builds use build input and even dynamically generate certain values (e.g. DDLM's CLSID) via build\override\dynamicdependencies-override.json

Any packages under test/* are by definition 'fake' as they cannot use Project Reunion published MSIXs, only local packages for F5/unit testing.

The 'real' Project Reunion MSIXs are used by Integration Tests, which by definition are built and run with artifacts of the end of the pipeline (the aggregator).